src/App.js

```diff
@@ -1,23 +1,14 @@
 import logo from './logo.svg';
 import './App.css';
+import PageFrame from "./Components/PageFrame";
+import {Route} from "react-router-dom";
+
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+        <Route component={} />
```

Comment:
You have to add path attribute and add a component that should be rendered in that path

Reply:
Also check the construction that should be used when declare routes